docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -36,6 +36,14 @@ CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
     [ LOCATION path ]
     [ TBLPROPERTIES ( key1=val1, key2=val2, ... ) ]
     [ AS select_statement ]
+
+row_format:    
+   : SERDE serde_class [WITH SERDEPROPERTIES (k1=v1, k2=v2, ...) ]
+   | DELIMITED [ FIELDS TERMINATED BY fields_termiated_char [ ESCAPED BY escaped_char] ] 
+               [ COLLECTION ITEMS TERMINATED BY collection_items_termiated_char ] 
+               [ MAP KEYS TERMINATED BY map_key_termiated_char ]
+               [ LINES TERMINATED BY row_termiated_char ]
+               [ NULL DEFINED AS null_char ]
```

Comment:
Not 100% sure though, is this indent correct? I looked around the the other syntax defs;
```
   | DELIMITED [ FIELDS TERMINATED BY fields_termiated_char [ ESCAPED BY escaped_char] ] 
<8 spaces>[ COLLECTION ITEMS TERMINATED BY collection_items_termiated_char ] 
<8 spaces>[ MAP KEYS TERMINATED BY map_key_termiated_char ]
<8 spaces>[ LINES TERMINATED BY row_termiated_char ]
<8 spaces>[ NULL DEFINED AS null_char ]
```
is more natural? cc: @huaxingao

Reply:
I am not 100% sure either, but seems to me it looks better with @GuoPhilipse's indent.
I also have something that doesn't strictly follow the indent rules. 
<img width="817" alt="Screen Shot 2020-07-16 at 6 44 52 PM" src="https://user-images.githubusercontent.com/13592258/87739364-88a7b180-c794-11ea-9088-56a55506561d.png">
I will probably do one more round of review tomorrow

Reply:
ok

Reply:
I checked other docs.  seems the indent rule is more general, i will check and follow it.